keep/api/models/alert.py

```diff
@@ -367,7 +367,10 @@ class EnrichAlertRequestBody(BaseModel):
 
 class BatchEnrichAlertRequestBody(BaseModel):
     enrichments: dict[str, str]
-    fingerprints: list[str]
+    fingerprints: Optional[list[str]] = None
+    cel: Optional[str] = None
+    limit: int = 1000
```

Comment:
Let's maybe remove these limit/offset parameters from batch enrichment request and hardcode limit:1000 offset:0
in the place where it's used now. I think no need to make limit and offset being part of public api especially because it's gonna be change in future.